### DISCUSSION TITLE
Welcome to Fulcrum Discussions!

### DISCUSSION THREAD

**Discussion by cculianu** [cculianu/Fulcrum#227] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Elshaffah** [comment]
Hi everyone. 
So I am El and I am a cryptocurrency enthusiast. The creation of Blockchain lead to my understanding of how a decentralized financial system should be like which is part of the reason I find it very interesting to become my own financial/bitcoin custodian by owning my private key. Bitcoin precisely introduced me to the world of dev and community of programmers working together for a common global purpose.
Though I am very much new to coding and each day I try as much as possible to sharpen my knowledge in the world of dev by utilizing open source projects. Hopefully I can connect with great minds in my journey of running a secure Bitcoin and lightning node.
